tests/python/unittest/test_runtime_heterogeneous.py

```diff
@@ -0,0 +1,405 @@
+# pylint: disable=too-many-locals
+"""Unit tests for heterogeneous runtime"""
+import json
+import numpy as np
+
+import tvm
+from tvm.contrib import graph_runtime, util
+import topi
+
+def get_simplex_graph(host_dev_type, device_dev_type):
+    r""" Return the hand-crafted json object where only one copy node is
+    inserted. Tis node copies data from the target device to cpu.
```

Comment:
Tis?

Reply:
@tmoreau89 Thanks. It was a typo. I was trying to say "This".